ci/build_cuda_cooperative_python.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euo pipefail
+cd "$(dirname "$0")/../python/cuda_cooperative"
```

Comment:
Note to self: maybe as a hack build the cuda_cccl wheels as well and upload them so that the test job can download and install them (duplicate this for cuda_parallel)